install_files/ansible-base/roles/common/tasks/remove_unused_packages.yml

```diff
@@ -7,6 +7,12 @@
     - apt
     - hardening
 
+# Remove the apparmor config file for snapd
+- name: Remove snapd config for apparmor
+  file:
+    path: /etc/apparmor.d/usr.lib.snapd.snap-confine.real
+    state: absent
+
```

Comment:
I just checked, and an `apt-get purge snapd` command removes this file even if it was already on disk. Since the removal above results in a purge, let's write a _test_ for absence of this file to ensure it's doing what we expect.